tool/data_dir_stat.py

```diff
@@ -0,0 +1,30 @@
+import os
+import datetime
+
+from katikati_pylib.logging import logging
+
+log = None
+
+def get_size_and_upload(CRYPTO_TOKEN_PATH, dirpath, project, firebase_client):
+    global log
+    if log is None:
+        log = logging.Logger(__file__, CRYPTO_TOKEN_PATH)
+    dirname = os.path.basename(dirpath)
+    dir_size = get_dir_size(dirpath)
+    entry = {
+        'project': project,
+        'size': round(dir_size, 1)
```

Comment:
consider using "size_in_mb" for clarity here as well

Reply:
Refactored it to that